doc/git_submodule.md

```diff
@@ -0,0 +1,70 @@
+# Using git submodules to develop a game
+
+Caveat: This is not necessarily the one true way. The framework and its practices
+are constantly being developed, but this document contains some pointers.
+
+If someone wants to add specific git commands here, please do. The current author
+will not bother, because formatting and anonymizing the repository URLs is too
+much work. This is the downside of open source, you do not always get everything
+for free.
```

Comment:
This paragraph could be replaced with a few words about why it's a good idea to use version control (eg. backup, pinpoint the introduction of a bug, easily switch between different states of a project, sync changes between multiple developers, etc.) and just point to the official getting started document: https://git-scm.com/book/en/v1/Getting-Started

Reply:
I apologize for not being in the best of moods when writing this. Some kind of fatigue seems to be hitting me, repeatedly. Anyway, yeah, a pointer to some kind of guide could be there.

Reply:
No worries. :slightly_smiling_face: If we do split this up into two or more documents, it is probably not needed anyway, since we can assume that the reader has the skills required to understand what they're reading, or else knows where to look for help.